Microsoft.Toolkit.Parsers/Markdown/MarkdownDocument.cs

```diff
@@ -96,6 +96,9 @@ internal static List<MarkdownBlock> Parse(string markdown, int start, int end, i
             int previousStartOfLine = start;
```

Comment:
The parse method seems really large, do we have a work item to split it up into other methods that we can more easily unit test?

Reply:
It would likely happen if we go through with #1767